### ISSUE TITLE
Replace deprecated functions

### ISSUE THREAD

**Issue by rustprooflabs** [rustprooflabs/pgdd#3]
In recent pgx versions I get the deprecation notice, e.g.

```bash
warning: use of deprecated item 'pgx::SpiHeapTupleData::get_datum': Please use the `by_ordinal` function instead
   --> src/lib.rs:253:29
    |
253 |                         row.get_datum(7), row.get_datum(8),
    |                             ^^^^^^^^^


```

**Comment by rustprooflabs** [comment]
Example usage: 
https://github.com/zombodb/pgx/blob/1d6af8bd6a51c2eeef751e57eecd1c9c3ec9017b/pgx-examples/bgworker/src/lib.rs#L65-L72

**Comment by rustprooflabs** [comment]
Resolved in #4
